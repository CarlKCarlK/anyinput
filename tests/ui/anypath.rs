use anyinput::anyinput;

#[anyinput]
pub fn any_str_len(s: AnyPath<usize>) -> Result<usize, anyhow::Error> {
    let len = s.len();
    Ok(len)
}

fn main() {}
