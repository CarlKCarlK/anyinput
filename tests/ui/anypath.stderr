error: AnyPath should not have a generic parameter, so 'AnyPath', not 'AnyPath<_>'.
 --> tests/ui/anypath.rs:4:23
  |
4 | pub fn any_str_len(s: AnyPath<usize>) -> Result<usize, anyhow::Error> {
  |                       ^^^^^^^^^^^^^^
